### PR TITLE
Added task to setup vscode unit test tab

### PR DESCRIPTION
**Pull request by CelianR**
### What does this PR do?

This adds a task to setup the unit test tab of vscode, an environment file should be created using the dummy environment used in the tests.

This allows to run easily a single test / module and to debug visually in one click.

Here is the [documentation page](https://datadoghq.atlassian.net/wiki/x/z4Jf6)

Screenshot 2024-07-12 at 17 03 23

### CODE REVIEW

File: tasks/vscode.py [hunk 1]
```diff
@@ -131,3 +131,35 @@ def setup_extensions(ctx: Context):
...
+        raise Exit(color_message('The .env file already exists', Color.RED))
+
+    with open('.env', 'w') as f:
+        for key, value in TEST_ENV.items():
+            print(f'{key}={value}', file=f)
+
+    print(color_message('The .env file has been created', Color.GREEN))
+
+    print(
+        f'''{color_message('note', Color.BLUE)}: You should add the following to your .vscode/settings.json file:
```
Comment (pducolin) [inline tasks/vscode.py:155]: **❓ question**
What if we did this when setting up the settings file? I think there is a task that was recently added to create the vs code settings file

Comment (pducolin) [reply tasks/vscode.py:155]: Maybe not, could not find it

**Review by pducolin — APPROVED** [review]

File: tasks/vscode.py (hunk 1 above)
Comment (pducolin) [reply tasks/vscode.py:155]: Ah here it is, it is open https://github.com/DataDog/datadog-agent/pull/27329 but could use this adds-on

Comment (CelianR) [reply tasks/vscode.py:155]: Good idea, I'll do it when this gets merged

**Review by chouetz — APPROVED** [review]

**Pull request merged**